cr/cmd/index.go

```diff
@@ -56,4 +57,7 @@ func init() {
 	flags.StringP("token", "t", "", "GitHub Auth Token (only needed for private repos)")
 	flags.StringP("git-base-url", "b", "https://api.github.com/", "GitHub Base URL (only needed for private GitHub)")
 	flags.StringP("git-upload-url", "u", "https://uploads.github.com/", "GitHub Upload URL (only needed for private GitHub)")
+	flags.String("pages-branch", "gh-pages", "The GitHub pages branch")
+	flags.Bool("push", false, "Push index.yaml to the GitHub Pages branch (must not be set if --pr is set)")
+	flags.Bool("pr", false, "Create a pull request for index.yaml against the GitHub Pages branch (must not be set if --push is set)")
```

Comment:
Nice 👍
Cross-referencing discussion in https://github.com/helm/chart-releaser-action/issues/33 about options design